src/components/CharacterList/style.module.css

```diff
@@ -0,0 +1,25 @@
+:root {
+  --gap-size: 2.5rem; 
+  --padding-size: 2.5rem;
+  --background-color: #1a242b;
+}
```

Comment:
it is better to keep all the variables together in the main css file or create a separate file like variables.css